files/scripts/syncd.sh

```diff
@@ -55,14 +55,14 @@ function wait_for_database_service()
 
 function getBootType()
```

Comment:
>getBootType [](start = 9, length = 11)

Can we reuse the function in `/usr/bin/syncd.sh`? If not, make sure you share the exactly same code snippet, add  code comments referring each other so we will not modify one without the other in future by mistake.